msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -18,8 +18,8 @@ The following schema sample shows all extensibility options:
 
 ```json
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.10/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.10",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
   "version": "1.0.0",
```

Comment:
please add defaultBlockUntilAdminAction and publisherDocsUrl to this sample schema. mention that publisherDocsUrl has to be htpps URL